bookkeeper-stats-providers/twitter-ostrich-provider/pom.xml

```diff
@@ -32,6 +32,38 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.libdir>${basedir}/lib</project.libdir>
   </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
Move the config of the plugin to the root pom and disable the plugin in bookkeeper-server